packages/types/src/primitive/Generic/Vote.ts

```diff
@@ -26,7 +32,7 @@ export default class Vote extends U8aFixed {
 
   private _conviction: Conviction;
 
-  public constructor (value?: any) {
+  public constructor (value?: InputTypes) {
```

Comment:
This might be a good idea for all type constructors actually, so we can remove the any

Reply:
Agreed. I'm on a drive to -

- expand tests (each time I touch them, I find bugs, aka check the VecFixed that just went it - it was buggy)
- cleanup CodeClimate, get rid of the duplication, get rid of the noise, make it code to be proud of

Well, I do these mostly while waiting for other stuff, just drives me batty that I'm still finding bugs atm.

Reply:
Tracking this over time - https://codeclimate.com/github/polkadot-js/api/trends/technical_debt